### PR TITLE
Readme: add link to convert meeting hour to local time zone

### PR DESCRIPTION
**Pull request by vgramer**
# Description

Readme: add a link to convert meeting hour to local time zone.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [X] Preview in PR

### CODE REVIEW

**Review by AbakusW — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Great for:
 
 Liqo is in an alpha version and not yet production-ready.
 
-Community Meeting: Monday 18.30 CET, 9.30 PST [here](https://meet.google.com/dyr-ieso-smu)
+Community Meeting: Monday 18.30 CET, 9.30 PST [here](https://meet.google.com/dyr-ieso-smu). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=18:30&tz=CET%20%28Central%20European%20Time%29)
```
Comment (palexster) [inline README.md:22]:
```suggestion
Community Meeting: Monday 18.30 CET, 9.30 PST [here](https://meet.google.com/dyr-ieso-smu). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=18:30&tz=CET%20%28Central%20European%20Time%29)
```

**Comment by palexster** [comment]
Hi @vgramer, thanks for this PR! It's ok for me! Can you squash your commits? I don't know if you noticed that the author of the commit is not linked with your account.

**Comment by vgramer** [comment]
@palexster I have squash commit and I haven't noticed my mail was not linked to my account. Thanks for noticing.

**Review by palexster — APPROVED** [review]

**Pull request merged**